src/selfie_msgs/action/driving.action

```diff
@@ -2,9 +2,7 @@
 bool mode
 ---
 #result definition
-
-float32 place_size 
```

Comment:
Dobrze by było zaktualizować wiki 
https://github.com/KNR-Selfie/selfie_carolocup2020/wiki/Messages-and-actions